locale/ar/docs/es6.md

```diff
@@ -0,0 +1,44 @@
+---
+title: ECMAScript 2015 (ES6) و ما بعدها
+layout: docs.hbs
+---
+# ECMAScript 2015 (ES6) و ما بعدها
+
+تم بناء الـ Node.js باستعمال نسخ حديثة من الـ [V8](https://developers.google.com/v8/)، و هذا يضمن اتاحة آخر المميزات الخاصة بالجافاسكريبت و الموافقة لـ [مواصفات JavaScript ECMA-262](http://www.ecma-international.org/publications/standards/Ecma-262.htm) للمطورين في الوقت المناسب، إضافة إلى التحسينات المستمرة في الأداء و الثبات.
```

Comment:
Update link #2138

Reply:
@Aissaoui-Ahmed Roger that. Changes has been made to fix issue #2138. thank you!
@InterstellarClub